cmd/lncli/commands.go

```diff
@@ -3173,10 +3174,36 @@ var forwardingHistoryCommand = cli.Command{
 			Name:  "max_events",
 			Usage: "the max number of events to return",
 		},
+		cli.StringFlag{
+			Name:  "since",
+			Usage: "number of H(ours)/D(ays)/W(eeks) before end_time (if negative) or after start_time or current time if no start_time specifued",
```

Comment:
This seems a bit too much functionality for one param.
I started giving it some thought and will probably add some proposal later.

In any case I think letters should be lower case, and there is a small typo in the last word: `specifued`

Reply:
I think if the param is called Interval and it is always positive than start_time and Interval results in end_time = start_time plus Interval; if end_time then start_time = end_time - interval. If only Interval, then end_time = present and start_time = present - Interval. How's that?

Reply:
Month is not one of the choices; I think Week is enough. Not sure what the problem with the rolling window is: just saying --interval 3W with no start or end time I think gives us that, in this case start time is three weeks from present and end time is present. Am I missing something?